[JBEAP-14635] Activate DatasourcesActiveCountTestCase

mackaypeter opened the pull request:
The PR with the fix  ( ironjacamar/ironjacamar#665 ) is now merged and ready for EAP 7.1.4, so the test case can be enabled.

mackaypeter committed:
[JBEAP-14635] Activate DatasourcesActiveCountTestCase

panossot commented on modules/src/main/java/org/jboss/additional/testsuite/jdkall/present/core/jca/DatasourcesActiveCountTestCase.java:
```diff
@@ -53,7 +53,7 @@
 @RunWith(Arquillian.class)
 @RunAsClient
-//@AdditionalTestsuite({"modules/testcases/jdkAll/Eap71x-Proposed/management/src/main/java#7.1.4","modules/testcases/jdkAll/Eap71x/management/src/main/java#7.1.4","modules/testcases/jdkAll/Wildfly/management/src/main/java#14.0.0"})
+@AdditionalTestsuite({"modules/testcases/jdkAll/Eap71x-Proposed/management/src/main/java#7.1.4","modules/testcases/jdkAll/Eap71x/management/src/main/java#7.1.4","modules/testcases/jdkAll/Wildfly/management/src/main/java#14.0.0"})
```
This should be @EapAdditionalTestsuite

mackaypeter commented:
@panossot I fixed the annotation, thanks!

panossot commented:
It seems fine. I will merge it when ironjacamar component version is updated on the server.

panossot commented:
Normally, it would be possible to merge it any time, because of versioning #7.1.4, but because of this commit : https://github.com/jbossas/jboss-eap7/commit/092ff50522f5f835f07faf2aafc832b4bd0b0993 (Develop towards EAP 7.1.4) this is not possible as the version of the server in now 7.1.4-x and the ironjacamar version is not updated till now.

mackaypeter commented:
Ok, thanks! Just wondering if it would be possible to somehow specify a non-snapshot version. That way we would be able to just merge away. The current way of merging seems like a lot of manual coordination.

panossot commented:
Yes, that would be possible. I will resolve https://github.com/jboss-set/eap-additional-testsuite/issues/44 when I have some time. Thx.

panossot commented:
@mackaypeter Hello, I have tested it again locally, and it seems the test fails. Can you double check?

panossot commented:
@mackaypeter I have tested with wildfly and the proposed 71x branch where ironjacamar is updated.

mackaypeter commented:
@jirka007 could you please take a look?

panossot commented:
@mackaypeter Can you update the PR to enable only for Eap71x-Proposed and Wildfly? (remove Eap71x for now). We will test it directly on CI, and if it fails I will remove it.

panossot commented:
@mackaypeter It works with eap71x-proposed 
https://thunder.sin2.redhat.com/jenkins/job/EAT-eap-7.1.x-proposed-build/233/console
Running org.jboss.additional.testsuite.jdkall.present.core.jca.DatasourcesActiveCountTestCase
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 7.692 sec - in org.jboss.additional.testsuite.jdkall.present.core.jca.DatasourcesActiveCountTestCase

Also could you update the PR adding Eap71x  (for Wildfly ironjacamar is not upgraded yet) (@EapAdditionalTestsuite({"modules/testcases/jdkAll/Eap71x-Proposed/core/src/main/java#7.1.4","modules/testcases/jdkAll/Eap71x/core/src/main/java#7.1.4"}))

I have enabled it only for eap71x-Proposed in order to test it on CI (@EapAdditionalTestsuite({"modules/testcases/jdkAll/Eap71x-Proposed/core/src/main/java#7.1.4"}))

mackaypeter commented:
@panossot I updated the PR. Only 71x and 71x-proposed are now included and I changed the path to use core instead of management. Hopefully it's ok now :)

panossot commented:
@mackaypeter thx
